core/tests/unit/km_core_keyboard_api.tests.cpp

```diff
@@ -56,10 +56,8 @@ TEST_F(KmCoreKeyboardApiTests, LoadFromBlobNull) {
   // Setup
   km::core::path kmxfile  = "";
 
-  std::unique_ptr<uint8_t> data(new uint8_t[0]);
-
   // Execute
-  auto status = km_core_keyboard_load_from_blob(kmxfile.stem().c_str(), data.get(), 0, &this->keyboard);
+  auto status = km_core_keyboard_load_from_blob(kmxfile.stem().c_str(), nullptr, 0, &this->keyboard);
```

Comment:
Why is this change in this PR?

Reply:
because the tests failed to compile in the container. Don't know why only in the container and not elsewhere, but that's why it's in this PR.

Reply:
ugh, that's not good.